### DISCUSSION TITLE
Raspberry Pi 64 support, please?

### DISCUSSION THREAD

**Discussion by gm8arv** [AlexandreRouma/SDRPlusPlus#414] [Ideas]
I hope there will be a Raspberry Pi version at some point.  I’ve been using the 64-bit OS on a Pi4 for tests and it seems to run most things, although I’ve not tested radio software!  I’m not a Linux expert but might be able to compile.  (I looked to see whether this had been discussed already, but didn’t find anything.)

**Comment by K0HYD** [comment]
Checkout “HamPi” a RPi image with Ham programs. He has installed SDR++ on it and you launch it from a menu. Works great here.  

73  
Dale

**Reply by gm8arv** [reply]
Thanks, Dale.  I have downloaded a copy to play with.

73,
David GM8ARV

**Comment by gm8arv** [comment]
Are you sure this is the 64-bit version of SDRpp?  I note that is is a chargeable product, and that the RasPi-4 isn't mentioned as supported.  RTL-SDR Blog used this:  https://sourceforge.net/projects/dragonos-pi64/files/

**Comment by the-gobleo-tinkerer** [comment]
Hi @gm8arv. Assuming you are using Raspberry Pi OS i.e. Raspbian, you could try building SDR++ yourself following the [build instructions](https://github.com/AlexandreRouma/SDRPlusPlus#building-on-linux--bsd), skipping the _Create a new root directory_ and _Running for development_ sections. I don't have a Pi at hand so I'm sorry I can't test for you, but as long as the Pi supports a recent enough OpenGL version it should build just fine.

**Reply by gm8arv** [reply]
Thanks, shuyuan-liu.  Yes, that is likely the approach I will take.  For some reason (seeing the Windows version first?) I had the impression this was a 64-bit only program.  Thanks for the link to the build instructions.

73, 
David GM8ARV

**Comment by K0HYD** [comment]
David,

I’m running it on a RaspberryPi 4B with 4GB of memory.

Here is a link to the WiKi: https://github.com/dslotter/HamPi/wiki.

Here is a link to the groups.io Forum: https://groups.io/g/Ham-Pi

You need to download the V1.5 image which was just released last week. It is the first one to have SDR++ on it.

Hope this helps!

73 de Dale - K0HYD


On October 8, 2021 at 6:52:36 AM, Shuyuan Liu ***@***.***) wrote:

Hi @gm8arv. Assuming you are using Raspberry Pi OS i.e. Raspbian, you could try building SDR++ yourself following the build instructions, skipping the Create a new root directory and Running for development sections. I don't have a Pi at hand so I'm sorry I can't test for you, but as long as the Pi supports a recent enough OpenGL version it should build just fine.
